internal/install/types/recipe.go

```diff
@@ -68,6 +74,18 @@ func (r *Recipe) HasApplicationTargetType() bool {
 	return r.HasTargetType(OpenInstallationTargetTypeTypes.APPLICATION)
 }
 
+func (r *Recipe) HasKeyword(keyword string) bool {
+	if len(r.Keywords) == 0 {
+		return false
+	}
+
+	for _, single := range r.Keywords {
+		return strings.EqualFold(single, keyword)
```

Comment:
This looks like it would only check the first keyword.

Reply:
Good catch, thanks @zlesnr